service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfigBuilder.java

```diff
@@ -129,6 +131,46 @@ public boolean isSsl() {
     return ipPortList;
   }
 
+  public ArrayList<IpPort> getRegistryIpPort() {
```

Comment:
This 3 methods and atract to one `public ArrayList<IpPort> getIpPort(String key) `